src/tests/end-to-end/tests/content/__snapshots__/guidance-content.test.ts.snap

```diff
@@ -1368,6 +1368,160 @@ exports[`Guidance Content pages rules/uniqueLandmark matches the snapshot 1`] =
 </DocumentFragment>
 `;
 
+exports[`Guidance Content pages test/adaptableContent/contrast/infoAndExamples matches the snapshot 1`] = `
+<DocumentFragment>
+  <div
+    class="content-container"
+  >
+    <div
+      class="content-left"
+    />
+    <div
+      class="content"
+    >
+      <h1>
+        Cannot find test/adaptableContent/contrast/infoAndExamples
```

Comment:
This snapshot looks off to me (this is probably why the e2e tests are failing)

Reply:
^ yes this was why but rebuilding fixed it. Just pushed a commit with the update and it should be good now!